### DISCUSSION TITLE
comparison with loopback.js

### DISCUSSION THREAD

**Discussion by sukeshlaghate** [marcj/deepkit#503] [Q&A]
How does Deepkit compare with loopback.js?

**Comment by marcus-sa** [comment]
In regards to what precisely?

**Reply by sukeshlaghate** [reply]
In regards to functionality. On the surface it looks like both are similar.

**Reply by marcj** [reply]
I don't know Loopback, but it seems it heavily relies in JavaScript functionalities like decorators to define runtime type information while Deepkit operates entirely on TypeScript types. So the difference is the amount of boilerplate and probably other stuff like performance + debug tools
